src/main/java/lolo/command/FindCommand.java

```diff
@@ -0,0 +1,23 @@
+package lolo.command;
+
+import lolo.Ui;
+import lolo.LoloException;
+import lolo.storage.Storage;
+import lolo.task.TaskList;
+
+/**
+ * A command to find tasks by a keyword in their description.
+ */
+class FindCommand extends Command {
+    private String keyword;
+
+    public FindCommand(String keyword) {
```

Comment:
Maybe add a method header here? You added method headers for all the other methods though, I think, so LGTM!